src/WebJobs.Extensions.DurableTask/Microsoft.Azure.WebJobs.Extensions.DurableTask.xml

```diff
@@ -2855,7 +2855,7 @@
         </member>
         <member name="T:Microsoft.Azure.WebJobs.Extensions.DurableTask.RequestMessage">
             <summary>
-            A message that represents an operation request or a lock request.
+            A message send to an entity, such as operation, signal, lock, release, or continue messages.
```

Comment:
This might not intentionally be part of your PR, but it looks like "send" should be "sent" here.

Reply:
Indeed.